CHANGES.rst

```diff
@@ -238,6 +238,10 @@ astropy.table
 astropy.tests
 ^^^^^^^^^^^^^
 
+- Removed ``--astropy-config-dir`` and ``--astropy-cache-dir`` options from
```

Comment:
Please move the entry to the API changes.

Reply:
Done and rebased.